### PR TITLE
Openstack IPI platform configuration

### PR DESCRIPTION
**Pull request by renan-campos**
To replace https://github.com/red-hat-storage/ocs-ci/pull/3031

### CODE REVIEW

File: ocs_ci/deployment/openstack.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+
+class OPENSTACKIPI:
+    """
+    A class to handle OpenStack IPI specific deployment
+    """
+
+    def __init__(self):
+        logger.info("OPENSTACK IPI")
+        # TODO: OCP,OCS deployment
```
Comment (clacroix12) [inline ocs_ci/deployment/openstack.py:13]: I take it this is just setting up the ground work for openstack IPI deployments. Should we expect the actual OCP/OCS deployment code in this PR or in a follow up?

Comment (renan-campos) [reply ocs_ci/deployment/openstack.py:13]: That may be addressed in a follow up PR. This is being added just to get the Tier1 tests to run.

Comment (clacroix12) [reply ocs_ci/deployment/openstack.py:13]: If that is the case then we may want to log that the deployment isn't actually happening, or raise an error here saying it isn't supported yet. That way we avoid confusion from anyone trying to deploy a cluster with openstack IPI.

**Comment by petr-balogh** [comment]
@rachael-george hello, can you please rebase the PR? So we can move on here and get it merged?  Just update also the log message as Coady suggested as deployment is not implemented. And open issue for the deployment implementation will be also good. Which you can mention in the comment in deployment code.

**Review by petr-balogh — APPROVED** [review]

**Pull request closed without merging**